test/e2e/data/e2e_conf.yaml

```diff
@@ -65,7 +65,7 @@ providers:
   - name: kubeadm
     type: BootstrapProvider
     files:
-      - sourcePath: "./shared/v1alpha4/metadata.yaml"
+      - sourcePath: "./shared/v1beta1/metadata.yaml"
```

Comment:
All the CAPI components need updating to v1.0.0-rc.0 if we want e2e tests to work.